assignments/arrays.js

```diff
@@ -63,31 +63,33 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log({ "id": 33, "car_make": "Jeep", "car_model": "Wrangler", "car_year": 2011 },);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = 50;
+console.log("car_make","Lincoln","car_model","Town Car",);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = ["car_model"];
+console.log(carModels.sort);
```

Comment:
Hmmm....where is the rest of the code on this one? You're on the right track with ultimately sorting the array in that manner. We need to iterate through the array of objects to grab the car models and push them to the carModels array.